lib/modules/versioning/conan/common.ts

```diff
@@ -3,7 +3,7 @@ import { regEx } from '../../../util/regex';
 
 export function makeVersion(
   version: string,
-  options: semver.Options
+  options: semver.RangeOptions
```

Comment:
I'm not sure this function is meant to work on ranges though, considering that the input is `version: string`.

Relevant PR: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/61658

Reply:
🤷‍♂️

Reply:
The upstream change was a straight rename. So:
1. Are we using it incorrectly?
2. Was the rename incorrect?

Reply:
it seems it was the wrong options interface before
![image](https://user-images.githubusercontent.com/1798109/185306719-6bfa3dc1-dc55-45db-b708-127b2c30e8b1.png)